pages/guideforms.py

```diff
@@ -55,6 +55,17 @@ def validate(self, value):
   'Write it from a web developer\'s point of view.\n\n'
   'Follow the example link below for more guidance.')
 
+# This email address pattern is more restrictive than actually allowed, but it covers most reasonable uses.
+EMAIL_ADDRESS_PATTERN = '[ ]*([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9\-\.]+)\.([a-zA-Z]{2,5}){1,25}[ ]*'
+
+EMAIL_ADDRESSES_PATTERN = '^' + EMAIL_ADDRESS_PATTERN + '(,' + EMAIL_ADDRESS_PATTERN + ')*$'
+
+MULTI_EMAIL_FIELD_ATTRS = {
+    'title':"Enter one or more comma-separated complete email addresses.",
+    'multiple': True, 
+    'placeholder': 'address1@domain.com, address2@chromium.org',
```

Comment:
I think the placeholder should use the word `user` rather than `address`, because the whole thing is the address rather than just that one part.